src/components/select/select.template.tsx

```diff
@@ -82,17 +82,16 @@ const SelectValue: FunctionalComponent<
     };
 
     return (
-        <div
+        <button
             class={anchorClassList}
             tabindex="0"
```

Comment:
I guess we could probably remove this `tabindex` too, now that the button is a proper `button`.

It shouldn't do any harm if it's left in, other than possibly being a source of confusion some day in the future 😅

Reply:
yeah. totally. no tabindex seems to be needed really. I didn't pay attention to it. And I don't know why MDC has chosen a `div` for this. It sounds uncannily weird.

Reply:
Added a fix here: https://github.com/Lundalogik/lime-elements/pull/1969